lib/WebDriverExpectedCondition.php

```diff
@@ -583,4 +584,65 @@ function (WebDriver $driver) use ($condition) {
             }
         );
     }
+
+    /**
+     * An expectation with the logical AND condition of the given conditions.
+     *
+     * @param WebDriverExpectedCondition[] $conditions Array of the conditions to be satisfied.
+     * @return static|false Array of the getApply() values of all satisfied conditions or FALSE if some condition
+     * failed.
+     */
+    public static function all($conditions)
+    {
+        return new static(
+            function (WebDriver $driver) use ($conditions) {
+                $results = [];
+                $allExist = true;
+
+                foreach ($conditions as $condition) {
+                    if (!$condition instanceof self) {
+                        throw new InvalidArgumentException(
+                            '$condition must be instance of WebDriverExpectedCondition class'
```

Comment:
```suggestion
                            $condition . ' must be instance of WebDriverExpectedCondition class'
```
This might work, you are not used double quotes so the value would not be added in the string

Reply:
This won't work. `$condition` is an object of `WebDriverExpectedCondition` class which doesn't have `__toString()` method